payroll/admin.py

```diff
@@ -55,8 +63,51 @@ def most_recent_job(self, obj):
         return obj.most_recent_job
 
 
+class PayrollSettingAdmin(SettingAdmin):
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
```

Comment:
i'm fine with the implementation choice, but I wonder if we might do this with a signal instead. seems like that might be good separation of concerns?

Reply:
I originally did do a signal in https://github.com/datamade/bga-payroll/pull/493/commits/357591ae9e1b1294beb6fa2cdd4e126ea4b49b21, but the admin has the big advantage of including the request, which I can use to get the scheme and host domain to build the index URL programmatically.